spec/main-process/atom-application.test.js

```diff
@@ -191,7 +191,7 @@ describe('AtomApplication', function () {
       const atomApplication = buildAtomApplication()
       const window1 = atomApplication.launch(parseCommandLine([path.join(tempDirPath, 'new-file')]))
       await focusWindow(window1)
-      await new Promise(resolve => setTimeout(resolve, 1000))
+      await new Promise(resolve => setTimeout(resolve, 2000))
```

Comment:
@nathansobo This is terrible, and fragile. Hoping you or someone else has a better idea. From what I can tell, this test may always have been brittle, we just never noticed anything because Atom was too slow? 🎱 

/cc @atom/feedback